src/worker/lambda/zygote/importCache.go

```diff
@@ -29,8 +30,9 @@ type ImportCache struct {
 // Sandbox death, etc)
 type ImportCacheNode struct {
 	// from config file:
-	Packages []string           `json:"packages"`
-	Children []*ImportCacheNode `json:"children"`
+	Packages        []string           `json:"packages"`
+	Children        []*ImportCacheNode `json:"children"`
+	SplitGeneration int                `json:"split_generation"`
```

Comment:
Not sure I see the value in including this in the deployment of the tree.  Isn't it an artifact of how you currently happen to generate a tree?

Reply:
"packages", "children" exists in original struct. "SplitGeneration" exists for print log message in warmup